roles/openshift_health_checker/openshift_checks/ram_diskspace_availability.py

```diff
@@ -0,0 +1,42 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+from openshift_checks.mixins import NotContainerizedMixin
+
+
+class RamDiskspaceAvailability(NotContainerizedMixin, OpenShiftCheck):
+    """Check that recommended memory and disk-space are available."""
+
+    name = "ram_diskspace_availability"
+    tags = ["preflight"]
+
+    # attempt to pull required docker images and fail if
+    # any images are missing, or error occurs during pull
+    def run(self, tmp, task_vars):
+        ansible_mounts = get_var(task_vars, "ansible_mounts")
+        diskfree = self.to_gigabytes(self.get_disk_space(ansible_mounts))
```

Comment:
> For containerized docker info would work for devicemapper w/ a thin pool, not sure about other backends.

Thanks, for now at least, I have added support for checking containerized installs that use devicemapper for storage driver